docs/recipes/WritingTests.md

```diff
@@ -8,12 +8,39 @@ hide_title: true
 
 # Writing Tests
 
-Because most of the Redux code you write are functions, and many of them are pure, they are easy to test without mocking.
+Because most of the Redux code you write are functions, and many of them are pure, they are easy to test without mocking. However, you should consider whether each piece of your Redux code needs it's own dedicated tests.
+
+:::tip
```

Comment:
:hand: Let's make boxes that contain links into `info` blocks instead of `tip` blocks

Reply:
Done